src/tests/integration/utils.py

```diff
@@ -146,6 +147,16 @@ def check_schema(self, response, schema, nullable_fields):
         def __get_schema_attributes():
             return schema['attributes']['properties']
 
+        # Validates returned attributes using pattern or format
```

Comment:
Can you move this comment into this function for the consistency with other codes? Example: https://github.com/osu-mist/express-api-skeleton/blob/master/src/tests/integration/integration_test.py#L65